frontend/app.py

```diff
@@ -0,0 +1,46 @@
+# %%
```

Comment:
Are we gonna have compatibility issues with the [frontend branch?](https://github.com/georgeamccarthy/protein_search/tree/fronted)

Reply:
Well, compatibility shouldn't be a problem but some care will be needed during the merge. It shouldn't be hard, anyways.